src/openforms/validations/drf_validators.py

```diff
@@ -22,18 +24,35 @@ class PluginValidator:
 
     requires_context = True
 
-    def __init__(self, plugin: str):
-        assert plugin in register, "Invalid plugin identifier specified"
-        self.plugin = plugin
+    def __init__(self, plugins: Iterable[str]):
```

Comment:
I wish we had a way to deal with https://github.com/python/typing/issues/256

Reply:
could've been `list[str]` but I followed your workshop advice :P